src/Generator/GeizhalsDE.php

```diff
@@ -223,7 +223,7 @@ private function buildRow($variation, KeyValue $settings)
                 'Herstellernummer'      => $variation['data']['variation']['model'],
                 'EAN'                   => $this->elasticExportCoreHelper->getBarcodeByType($variation, $settings->get('barcode')),
                 'Kategorie'             => $this->elasticExportCoreHelper->getCategory((int)$variation['data']['defaultCategories'][0]['id'], $settings->get('lang'), $settings->get('plentyId')),
-                'Grundpreis'            => $this->elasticExportPriceHelper->getBasePrice($variation, $priceList['price'], $settings->get('lang'), '/', false, true),
+                'Grundpreis'            => $this->elasticExportPriceHelper->getBasePrice($variation, (float)$priceList['price'], $settings->get('lang'), '/', false, true),
```

Comment:
Fits, but what happens with the condition in line 212? To be save, let us please also add there for the price a float cast.

Reply:
ok, fixed